src/Microsoft.OData.Client/Serialization/HttpWebRequestMessage.cs

```diff
@@ -8,204 +8,196 @@ namespace Microsoft.OData.Client
 {
     using System;
     using System.Collections.Generic;
-    using System.Diagnostics;
-    using System.Diagnostics.CodeAnalysis;
+    using Microsoft.OData.Client;
     using System.IO;
-    using System.Globalization;
+    using System.Linq;
     using System.Net;
-#if PORTABLELIB
-    using System.Reflection;
-    using System.Xml;
-#endif
+    using System.Net.Http;
+    using System.Net.Http.Headers;
+    using System.Threading.Tasks;
     using Microsoft.OData;
+    using System.Globalization;
+    using System.Text;

```

Comment:
nitpick: order the using statements. I think it's a standard to start with the System.* namespaces before other namespaces. And order should be ideally alphabetical.